tools/dissector/sample_dissector.cpp

```diff
@@ -196,11 +195,67 @@ namespace OpenDDS
       return ns_stack_.back();
     }
 
-    void Sample_Base::add_protocol_field(enum ftenum ft, field_display_e fd) {
-      Field_Context * fc = get_context();
-      Sample_Manager::instance().add_protocol_field(
-        &fc->hf_, get_ns(), fc->label_, ft, fd
-      );
+    void Sample_Base::push_idl_name(std::string name) {
+      Sample_Base::clear_ns();
+
+      // If name begins with "IDL:", remove it
+      const std::string idl_prefix("IDL:");
```

Comment:
Why not use find?

Reply:
I didn't write this recently, just moved it to it's own function, but I think it's because it's just looking for "IDL:" at the beginning of the string only.

Reply:
I agree that it would be cleaner to use find() (comparing result to 0).  Might as well clean it up now that we're looking at it.